client/src/docs-ui/choose-integration-anchor/choose-integration-anchor.tsx

```diff
@@ -27,7 +27,11 @@ export class DocsChooseIntegrationAnchor {
                 this.page && `${this.page.route}/q/integration/${filterValue}`;
 
               return (
-                <docs-card key={label} url={route}>
+                <docs-card
+                  key={label}
+                  url={route}
+                  QSPs={`?sc_icampaign=${filterValue}-start&sc_ichannel=choose-integration`}
```

Comment:
should `QSPs` be `qsps` ?

Reply:
It's only `qsps` instead of `QSPs` in snapshots and HTML and such, not JSX, as far as I can tell... in any case, it works in testing.